build.gradle

```diff
@@ -55,7 +55,10 @@ subprojects {
         dockerFile {
             dockerFile = file("$workingDir/Dockerfile")
             dockerBuild {
-                repository = "xeniteu/alfresco-share-" + "${project.share.flavor}"
+                if (!(project.hasProperty("repoName"))) {
+                    ext.repoName = "hub.xenit.eu"
```

Comment:
Default to `xeniteu` on docker hub instead?

Reply:
Don't know, enterprise images are still most important ones for Xenit and they are built on Jenkins and pushed to hub.xenit.eu.